package.json

```diff
@@ -55,6 +55,8 @@
     "dotenv": "5.0.0",
     "dotenv-expand": "4.2.0",
     "electron": "2.0.1",
+    "electron-debug": "^2.0.0",
+    "electron-devtools-installer": "^2.2.4",
```

Comment:
Nit: Could you remove the minor-matching caret? 

We have a Yarn lockfile so it's not _super_ important, but if we ever have to rebuild the lockfile, I like the safety of knowing our dependencies themselves are version-specific.